### ISSUE TITLE
Star imports from `v2/__init__.py` and `v3/__init__.py` fail

### ISSUE THREAD

**Issue by henribru** [googleads/google-ads-python#260]
`from google.ads.google_ads.v2 import *` and `from google.ads.google_ads.v3 import *` fail with `AttributeError: unknown sub-module 'Enums'.` This happens because `__all__` is defined to contain uppercase versions of all the keys in `_lazy_name_to_package_map`. This works for the services since there is a corresponding uppercase type for each snake case module, but the dict also contains `types` and `enums`, for which this is not the case.

**Comment by BenRKarl** [comment]
@henribru we don't intend for the modules under each version to be imported directly, those objects should be retrieved using the `get_type` and `get_service` client methods. You can see usages throughout our [examples](https://github.com/googleads/google-ads-python/tree/master/examples). All the code under the version module is generated, and it's possible that the structure could change, which is why we have the getter methods to abstract that structure away.

Is there a specific use-case you have that necessitates `import *`? We could consider improvements to the client to allow retrieval of multiple classes at once.

**Comment by henribru** [comment]
@BenRKarl I don't really need it, no, this is just something I noticed while working on type stubs for the library. But it's clearly a bug to put something in `__all__` that's not actually importable (it defeats the purpose of `__all__`), so I'm not sure how you can consider it working as intended.

**Comment by BenRKarl** [comment]
@henribru Good point. I looked more closely at these files and the `_lazy_name_to_package_map` maps classnames to filenames and module paths so you can retrieve a class like:

```
from google.ads.google_ads.v3.services.google_ads_service import GoogleAdsService
```

However this pattern doesn't apply to `enums` and `types` as in there is no class at this path:

```
from google.ads.google_ads.v3.services.enums import Enums
```

So I changed this to a bug and will be posting a PR shortly, thanks for pointing it out.

**Comment by BenRKarl** [comment]
This is fixed in `5.0.3` of the library.
